tests/scriptlets/prevent-refresh.test.js

```diff
@@ -65,6 +85,37 @@ test('Prevent redirect, delay from arg', (assert) => {
     setTimeout(() => {
         assert.ok(location.href.indexOf('prevent-refresh') !== -1, 'Redirect prevented');
         assert.strictEqual(window.hit, 'FIRED', 'hit function was executed');
+        removeMeta();
+        done();
+    }, 1 * 1000);
+});
+
+test('Prevent redirect, in case of invalid content, checks for - TypeError: Reduce of empty array with no initial value', (assert) => {
+    const REL_PAGE_PATH = './test-files/empty.html';
+    const contentValue = `invalid;url=${REL_PAGE_PATH}`;
+    let testPassed = true;
+    createMeta(contentValue);
+
+    // Check if there is a "Reduce of empty array with no initial value" error in console
+    // if so, then set "testPassed" to "false"
+    const checkConsole = () => {
+        const wrapperLog = (target, thisArg, args) => {
+            if (args[0]?.message?.includes('Reduce of empty array with no initial value')) {
+                testPassed = false;
+            }
+            return Reflect.apply(target, thisArg, args);
+        };
+        const handlerLog = {
+            apply: wrapperLog,
+        };
+        window.console.log = new Proxy(window.console.log, handlerLog);
+    };
+    checkConsole();
+    runScriptlet(name);
+    const done = assert.async();
+    setTimeout(() => {
+        assert.ok(testPassed, 'No error');
+        removeMeta();
```

Comment:
could you please add checking for `hit`? it should be `undefined` for this test, right?

Reply:
Done.